CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Changelog
 
-## Unreleased
+## 0.14.0 / 2022-11-2
```

Comment:
I think the script failed more because of the date format @sjcobb .

```suggestion
## 0.14.0 / 2022-11-02
```

Reply:
> I think the script failed more because of the date format @sjcobb .

😆 oh dang, thanks!